LibAddonMenu-2.0/LibAddonMenu-2.0/controls/editbox.lua

```diff
@@ -21,6 +22,14 @@ if not LAM:RegisterWidget("editbox", widgetVersion) then return end
 
 local wm = WINDOW_MANAGER
 
+local function GetValidTextType(textType)

```

Comment:
Don't think this belongs into this pull request?

Reply:
Seems it was something from Scootworks addititions... Will strip it